ui/admin/app/abilities/scope.js

```diff
@@ -21,6 +21,7 @@ export default class OverrideScopeAbility extends OverrideModelAbility {
   get canCreate() {
     switch (this.collection) {
       case 'storage-buckets':
+      case 'policies':
```

Comment:
Should this be called `policies` or `storage-policies`?

Reply:
the API calls it `policies` so I kept it to be consistent